lib/sortBy/sortBy.test.js

```diff
@@ -20,7 +20,7 @@ test('can compare primitives', () => {
   expect(sort(arrayOfBooleans)).toEqual([false, false, false, true, true])
 })
 
-test('doesn\'t mutate original array', () => {
+test("doesn't mutate original array", () => {
```

Comment:
use " or ' for consistency

Reply:
Prettier adds double quotes if string contains single